src/script/script.h

```diff
@@ -583,4 +595,15 @@ struct CScriptWitness
     std::string ToString() const;
 };
 
+class CReserveScript
```

Comment:
You've accidentally reintroduced this in ebcfe3584199f281f825625dcadef8535dece536

Reply:
What do you mean? isn't https://github.com/bitcoin/bitcoin/commit/ebcfe3584199f281f825625dcadef8535dece536 the only thing where this exists?